### PR TITLE
Add documentation for Custom timeframe on Monitors

### PR DESCRIPTION
**Pull request by NBParis**
### What does this PR do?
Update the time window limits documented so users know what they can input when setting the 

### Motivation
Delivery of the custom timeframe for monitors

### Preview

https://docs-staging.datadoghq.com/nils/monitor-custom-timeframe/api/v1/monitors/#create-a-monitor
https://docs-staging.datadoghq.com/nils/monitor-custom-timeframe/monitors/monitor_types/

Monitor pages changed:

- Metrics
- Event
- live Process
- Outliers
- Forecast
- Anomalies
- RUM
- APM

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/monitor_types/metric.md [hunk 1]
```diff
@@ -126,8 +126,8 @@ The alert conditions vary slightly based on the chosen detection method.
 
 * The `average`, `maximum`, `minimum`, or `in total`
 * of the `change` or `% change`
-* over `5 minutes`, `15 minutes`, `1 hour`, etc.
-* compared to `5 minutes`, `15 minutes`, `1 hour`, etc. before
+* over `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48h
```
Comment (ruthnaebeck) [inline content/en/monitors/monitor_types/metric.md:129]:
```suggestion
* over `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48 hours.
```

File: content/en/monitors/monitor_types/metric.md [hunk 2]
```diff
@@ -126,8 +126,8 @@ The alert conditions vary slightly based on the chosen detection method.
...
+* compared to `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48h before
```
Comment (ruthnaebeck) [inline content/en/monitors/monitor_types/metric.md:130]:
```suggestion
* compared to `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 48 hours before.
```

File: content/en/monitors/monitor_types/outlier.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ The outlier monitor requires a metric with a group (hosts, availability zones, p
 ### Set alert conditions
 
 * Trigger a separate alert for each outlier `<GROUP>`
-* during the last `5 minutes`, `15 minutes`, `1 hour`, etc.
+* during the last `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 24h
```
Comment (ruthnaebeck) [inline content/en/monitors/monitor_types/outlier.md:38]:
```suggestion
* during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 24 hours.
```

File: content/en/monitors/monitor_types/process.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ After defining your search, a graph is displayed above the search inputs with an
 ### Set alert conditions
 
 * The count was `above`, `above or equal to`, `below`, or `below or equal to`
-* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc.
+* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48h
```
Comment (ruthnaebeck) [inline content/en/monitors/monitor_types/process.md:44]:
```suggestion
* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 48 hours.
```

File: content/en/monitors/monitor_types/real_user_monitoring.md [hunk 5]
```diff
@@ -37,7 +37,7 @@ As you define the search query, the graph above the search fields updates.
 ### Set alert conditions
 
 * Trigger when the metric is `above`, `above or equal to`, `below`, or `below or equal to`
-* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc.
+* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 5 minutes and 24h
```
Comment (ruthnaebeck) [inline content/en/monitors/monitor_types/real_user_monitoring.md:40]:
```suggestion
* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 5 minutes and 24 hours.
```

File: data/api/v1/translate_actions.json [hunk 6]
```diff
@@ -938,4 +938,4 @@
     "description": "Disables muting all monitors. Throws an error if mute all\nwas not enabled previously.",
     "summary": "Unmute all monitors"
   }
-}
\ No newline at end of file
+}
```
Comment (ruthnaebeck) [inline data/api/v1/translate_actions.json:941]: I think this page has to be changed through the API spec repo.

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

File: data/api/v1/translate_actions.json (hunk 6 above)
Comment (NBParis) [reply data/api/v1/translate_actions.json:941]: Do you mind pointing me where this is? Happy to make the changes there but couldn't find it in the documentation repo.

Comment (kayayarai) [reply data/api/v1/translate_actions.json:941]: @NBParis https://github.com/DataDog/datadog-api-spec/blob/master/spec/v1/monitor.yaml

File: content/en/monitors/monitor_types/metric.md [hunk 7]
```diff
@@ -126,8 +126,8 @@ The alert conditions vary slightly based on the chosen detection method.
 
 * The `average`, `maximum`, `minimum`, or `in total`
 * of the `change` or `% change`
-* over `5 minutes`, `15 minutes`, `1 hour`, etc.
-* compared to `5 minutes`, `15 minutes`, `1 hour`, etc. before
+* over `5 minutes`, `15 minutes`, `1 hour`, ... or `custom` to set a value between 1 minute and 48 hours.
```
Comment (kayayarai) [inline content/en/monitors/monitor_types/metric.md:129]:
```suggestion
* over `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 48 hours.
```
missed one

File: data/api/v1/translate_actions.json (hunk 6 above)
Comment (NBParis) [reply data/api/v1/translate_actions.json:941]: Opened a PR and will check with engineering on making it through: https://github.com/DataDog/datadog-api-spec/pull/698

**Comment by NBParis** [comment]
PR to make the API changes: https://github.com/DataDog/datadog-api-spec/pull/698

**Review by kayayarai — APPROVED** [review]

**Pull request merged**